src/components/experiment/Experiment.tsx

```diff
@@ -0,0 +1,130 @@
+import React, { useEffect, useState } from 'react';
+import ListGroup from 'react-bootstrap/ListGroup';
+import Row from 'react-bootstrap/Row';
+import Col from 'react-bootstrap/Col';
+import Button from 'react-bootstrap/Button';
+import Spinner from 'react-bootstrap/Spinner';
+import Toast from 'react-bootstrap/Toast';
+import Container from 'react-bootstrap/Container';
+import ExperimentItem from './ExperimentItem';
+
+function generateItems(experimentUrls: Array<string>) {
+  const items: Array<JSX.Element> = [];
+  experimentUrls.forEach(url => {
+    items.push(<ExperimentItem url={url} key={url} />);
+  });
+
+  return items;
+}
```

Comment:
May as well use an arrow function like you do for the other methods in this file for consistency.

Reply:
Refactored in eb6d5da